tests/beaker_tests/tacacs_server_group/tacacs_server_group_provider_defaults.rb

```diff
@@ -62,18 +62,14 @@
 test_name "TestCase :: #{testheader}" do
   # @step [Step] Sets up switch for provider test.
   step 'TestStep :: Setup switch for provider' do
-    on(master, TacacsServerGroupLib.create_tacacs_server_group_manifest_absent)
-
-    # Expected exit_code is 2 since this is a puppet agent cmd with change.
-    cmd_str = get_namespace_cmd(agent, PUPPET_BINPATH +
-      'agent -t', options)
-    on(agent, cmd_str, acceptable_exit_codes: [0, 2])
+    resource_absent_cleanup(agent, 'tacacs_server')
+    resource_absent_cleanup(agent, 'tacacs_server_group')
 
     cmd_str = get_vshell_cmd('conf t ; tacacs-server host 2.2.2.2')
     on(agent, cmd_str)
     cmd_str = get_vshell_cmd('conf t ; tacacs-server host 3.3.3.3')
     on(agent, cmd_str)
-    cmd_str = get_vshell_cmd('conf t ; tacacs-server host 4.4.4.4')
+    cmd_str = get_vshell_cmd('conf t ; tacacs-server host 2004::44')
```

Comment:
We are trying to move away from using vsh commands to configure the device.  This does not work well across our various Cisco platforms.  Can we rework this setup section to use `tacacs_server` or `cisco_tacacs_server_host` instead?